app/models/vote.rb

```diff
@@ -0,0 +1,7 @@
+class Vote < ApplicationRecord
+  belongs_to :review
+  belongs_to :user
+
+  validates :upvotes, presence: true, numericality: { in: 0..1 }
+  validates :downvotes, presence: true, numericality: { in: 0..1 }
```

Comment:
not bad to explicitly add `presence: true`, but worth noting that `numericallity` _assumes_ presence true! If we wanted it to be optional, we would have to add to numericality `allow_nil` rather than a validation for presence